enrollment-server/src/main/resources/application.properties

```diff
@@ -76,80 +76,6 @@ powerauth.service.security.clientSecret=
-enrollment-server.onboarding-process.otp.max-failed-attempts=5
-enrollment-server.onboarding-process.otp.resend-period=PT30S
-enrollment-server.onboarding-process.activation.expiration.seconds=300
-enrollment-server.onboarding-process.verification.expiration.seconds=86400
-enrollment-server.onboarding-process.max-processes-per-day=5
-
-# Identity Verification Configuration
-enrollment-server.identity-verification.enabled=false
-enrollment-server.identity-verification.data-retention.hours=1
-enrollment-server.identity-verification.expiration.seconds=300
-enrollment-server.identity-verification.otp.enabled=true
-
-# Provider Configuration
-#enrollment-server.document-verification.provider=zenid
-enrollment-server.document-verification.provider=mock
-enrollment-server.document-verification.cleanupEnabled=false
-enrollment-server.document-verification.checkInProgressDocumentSubmits.cron=0/5 * * * * *
-enrollment-server.document-verification.verificationOnSubmitEnabled=true
-enrollment-server.document-verification.checkDocumentsVerifications.cron=0/5 * * * * *
-enrollment-server.document-verification.checkDocumentSubmitVerifications.cron=0/5 * * * * *
-
-enrollment-server.presence-check.enabled=true
-#enrollment-server.presence-check.provider=iproov
-enrollment-server.presence-check.provider=mock
-enrollment-server.presence-check.cleanupEnabled=false
-# Enables/disabled verification of the presence check selfie photo with the documents
-enrollment-server.presence-check.verifySelfieWithDocumentsEnabled=false
-
-# iProov configuration
-enrollment-server.presence-check.iproov.apiKey=${IPROOV_API_KEY}
-enrollment-server.presence-check.iproov.apiSecret=${IPROOV_API_SECRET}
-enrollment-server.presence-check.iproov.assuranceType=${IPROOV_ASSURANCE_TYPE:genuine_presence}
-enrollment-server.presence-check.iproov.ensureUserIdValueEnabled=${IPROOV_ENSURE_USER_ID_VALUE_ENABLED:false}
-enrollment-server.presence-check.iproov.riskProfile=${IPROOV_RISK_PROFILE:}
-enrollment-server.presence-check.iproov.serviceBaseUrl=${IPROOV_SERVICE_BASE_URL}
-enrollment-server.presence-check.iproov.serviceHostname=${IPROOV_SERVICE_HOSTNAME}
-enrollment-server.presence-check.iproov.serviceUserAgent=Wultra/EnrollmentServer
-
-# iProov REST client configuration
-enrollment-server.presence-check.iproov.restClientConfig.acceptInvalidSslCertificate=false
-enrollment-server.presence-check.iproov.restClientConfig.maxInMemorySize=10485760
-enrollment-server.presence-check.iproov.restClientConfig.proxyEnabled=false
-enrollment-server.presence-check.iproov.restClientConfig.proxyHost=
-enrollment-server.presence-check.iproov.restClientConfig.proxyPort=0
-enrollment-server.presence-check.iproov.restClientConfig.proxyUsername=
-enrollment-server.presence-check.iproov.restClientConfig.proxyPassword=
-
-spring.security.oauth2.client.provider.app.token-uri=http://localhost:6060/oauth/token
-
-# ZenID configuration
-enrollment-server.document-verification.zenid.apiKey=${ZENID_API_KEY}
```

Comment:
So this belongs to the downstream project?

Reply:
I see, I have to expand, it is here https://github.com/wultra/enrollment-server/pull/235/files#diff-53ea8234fd009b798ac2ab82ae426290ba589a47313322d4215a7728152dedd2L129

Reply:
Yes, this has nothing to do with downstream, just a bit of mess due to duplicit configuration and older property names as left overs.